README.md

```diff
@@ -1,15 +1,24 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-
+ Javascript Capstone 
 > Description the project.
 
```

Comment:
- [ ] For proper documentation, kindly change this dummy title to something more descriptive of the project you just built.

 - [ ] Kindly add a more elaborate paragraph describing the project. 'Javascript Capstone' is not particularly descriptive.